client/app/bundles/course/assessment/question/programming/components/OnlineEditorPythonView.jsx

```diff
@@ -351,7 +358,7 @@ class OnlineEditorPythonView extends React.Component {
                   <FlatButton
                     label={this.props.intl.formatMessage(translations.addNewTestButton)}
                     icon={<i className="fa fa-plus" />}
-                    disabled={this.props.isLoading}
+                    disabled={this.props.isLoading || numAllTestCases >= 99}
```

Comment:
Shall we extract the 99 into a const left at the top of the file? Easier to modify in the future.